### PR TITLE
Moved redundant codes in helper class.

### PR DESCRIPTION
**Pull request by mdanish98**
Dear Frank,

Added helper classes for MqttAASRegistry, MqttAASAggregator, MqttSubmodelAPI and moved redundant codes in these classes.

Thanks and Regards
Mohammad Ghazanfar Ali Danish

### CODE REVIEW

File: src/main/java/org/eclipse/basyx/extensions/aas/aggregator/mqtt/MqttAASAggregator.java [hunk 1]
```diff
@@ -35,9 +35,7 @@
 	private static Logger logger = LoggerFactory.getLogger(MqttAASAggregator.class);
 
 	// List of topics
```
Comment (FrankSchnicke) [inline src/main/java/org/eclipse/basyx/extensions/aas/aggregator/mqtt/MqttAASAggregator.java:37]: Please remove this now obsolete comment

File: src/main/java/org/eclipse/basyx/extensions/aas/aggregator/mqtt/MqttAASAggregatorHelper.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ ******************************************************************************/
+package org.eclipse.basyx.extensions.aas.aggregator.mqtt;
+
+/**
+ * A helper class for MqttAASAggregator.
```
Comment (FrankSchnicke) [inline src/main/java/org/eclipse/basyx/extensions/aas/aggregator/mqtt/MqttAASAggregatorHelper.java:13]: Please be a little bit more specific here, i.e., "A helper class containing string constants of topics used by the AASAggregator"

File: src/main/java/org/eclipse/basyx/extensions/aas/registration/mqtt/MqttAASRegistryHelper.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ ******************************************************************************/
+package org.eclipse.basyx.extensions.aas.registration.mqtt;
+
+import org.eclipse.basyx.submodel.metamodel.api.identifier.IIdentifier;
+
+/**
+ * A helper class for MqttAASRegistry.
```
Comment (FrankSchnicke) [inline src/main/java/org/eclipse/basyx/extensions/aas/registration/mqtt/MqttAASRegistryHelper.java:15]: See comment MqttAASAggregatorHelper

File: src/main/java/org/eclipse/basyx/extensions/submodel/mqtt/MqttSubmodelAPIHelper.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ ******************************************************************************/
+package org.eclipse.basyx.extensions.submodel.mqtt;
+
+/**
+ * A helper class for MqttSubmodelAPI.
```
Comment (FrankSchnicke) [inline src/main/java/org/eclipse/basyx/extensions/submodel/mqtt/MqttSubmodelAPIHelper.java:13]: See comment MqttAASAggregatorHelper

File: src/main/java/org/eclipse/basyx/extensions/aas/registration/mqtt/MqttAASRegistryHelper.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+ *
+ */
+public class MqttAASRegistryHelper {
+	public static final String TOPIC_REGISTERAAS = "BaSyxRegistry_registeredAAS";
+	public static final String TOPIC_REGISTERSUBMODEL = "BaSyxRegistry_registeredSubmodel";
+	public static final String TOPIC_DELETEAAS = "BaSyxRegistry_deletedAAS";
+	public static final String TOPIC_DELETESUBMODEL = "BaSyxRegistry_deletedSubmodel";
+	
+	/**
+	 * Concatenates Asset Administration Shell Id and SmId.
```
Comment (FrankSchnicke) [inline src/main/java/org/eclipse/basyx/extensions/aas/registration/mqtt/MqttAASRegistryHelper.java:27]: Please use a more descriptive name here - The intent of the function is to create a message payload representing a descriptor change of a submodel of a specific AAS. This should also reflect in the method comment.
Thus, a better name would be "createSubmodelDescriptorOfAASChangedPayload"

**Review by FrankSchnicke — CHANGES_REQUESTED** [review]

File: src/main/java/org/eclipse/basyx/extensions/aas/aggregator/mqtt/MqttAASAggregator.java (hunk 1 above)
Comment (mdanish98) [reply src/main/java/org/eclipse/basyx/extensions/aas/aggregator/mqtt/MqttAASAggregator.java:37]: Removed this obsolete comment

File: src/main/java/org/eclipse/basyx/extensions/aas/aggregator/mqtt/MqttAASAggregatorHelper.java (hunk 2 above)
Comment (mdanish98) [reply src/main/java/org/eclipse/basyx/extensions/aas/aggregator/mqtt/MqttAASAggregatorHelper.java:13]: Updated the class description.

File: src/main/java/org/eclipse/basyx/extensions/aas/registration/mqtt/MqttAASRegistryHelper.java (hunk 3 above)
Comment (mdanish98) [reply src/main/java/org/eclipse/basyx/extensions/aas/registration/mqtt/MqttAASRegistryHelper.java:15]: Updated the class description.

File: src/main/java/org/eclipse/basyx/extensions/submodel/mqtt/MqttSubmodelAPIHelper.java (hunk 4 above)
Comment (mdanish98) [reply src/main/java/org/eclipse/basyx/extensions/submodel/mqtt/MqttSubmodelAPIHelper.java:13]: Updated the class description.

File: src/main/java/org/eclipse/basyx/extensions/aas/registration/mqtt/MqttAASRegistryHelper.java (hunk 5 above)
Comment (mdanish98) [reply src/main/java/org/eclipse/basyx/extensions/aas/registration/mqtt/MqttAASRegistryHelper.java:27]: Updated the method description and method name.

**Review by FrankSchnicke — APPROVED** [review]

**Pull request merged**